resources/deliberate-ub.md

```diff
@@ -19,6 +19,8 @@ We should evaluate whether there truly is some use-case here that is not current
   see the [discussion here](https://github.com/rust-lang/unsafe-code-guidelines/issues/449).<br>
   The alternative is to not use the "fast path" for problematic types (and fall back to the SeqLock), but that requires some way to query at `const`-time whether the type contains padding (or provenance).
   (Or of course one can use inline assembly, but it would be better if that was not required.)
+* crossbeam's deque uses [volatile accesses that really should be atomic instead](https://github.com/crossbeam-rs/crossbeam/blob/5a154def002304814d50f3c7658bd30eb46b2fad/crossbeam-deque/src/deque.rs#L70-L88).
+  They cannot use atomic accesses as those are not possible for arbitrary `T`.
```

Comment:
@taiki-e do you know if atomic bytewise memcpy (https://github.com/rust-lang/rfcs/pull/3301) would solve this problem? It sure looks like it would.

Reply:
Yes, I mentioned the use in crossbeam-deque in that RFC over 2 years ago.

- https://github.com/rust-lang/rfcs/pull/3301#discussion_r945408359
- https://github.com/rust-lang/rfcs/pull/3301#discussion_r945429538